pkg/ocicni/ocicni.go

```diff
@@ -261,34 +261,34 @@ func loadNetworks(exec cniinvoke.Exec, confDir string, binDirs []string) (map[st
 		if strings.HasSuffix(confFile, ".conflist") {
 			confList, err = libcni.ConfListFromFile(confFile)
 			if err != nil {
-				logrus.Warningf("Error loading CNI config list file %s: %v", confFile, err)
+				log().Warnf("loading CNI config list from file %s: %v", confFile, err)
 				continue
 			}
 		} else {
 			conf, err := libcni.ConfFromFile(confFile)
 			if err != nil {
-				logrus.Warningf("Error loading CNI config file %s: %v", confFile, err)
+				log().Warnf("loading CNI config from file %s: %v", confFile, err)
 				continue
 			}
 			if conf.Network.Type == "" {
-				logrus.Warningf("Error loading CNI config file %s: no 'type'; perhaps this is a .conflist?", confFile)
+				log().Warnf("loading CNI config file %s: no 'type'; perhaps this is a .conflist?", confFile)
 				continue
 			}
 			confList, err = libcni.ConfListFromConf(conf)
 			if err != nil {
-				logrus.Warningf("Error converting CNI config file %s to list: %v", confFile, err)
+				log().Warnf("converting CNI config file %s to list: %v", confFile, err)
 				continue
 			}
 		}
 		if len(confList.Plugins) == 0 {
-			logrus.Warningf("CNI config list %s has no networks, skipping", confFile)
+			log().Warnf("CNI config list %s has no networks, skipping", confFile)
 			continue
 		}
 		if confList.Name == "" {
 			confList.Name = path.Base(confFile)
 		}
 
-		logrus.Infof("Found CNI network %s (type=%v) at %s", confList.Name, confList.Plugins[0].Network.Type, confFile)
+		log().Debugf("found CNI network %s (type=%v) at %s", confList.Name, confList.Plugins[0].Network.Type, confFile)
```

Comment:
Any reason this switched from Infof() to Debugf() in this PR? At least until one of the reload PRs lands, this shouldn't be called often and it's very useful information.

Reply:
Okay, I will revert it back to `Infof`